trackpy/feature.py

```diff
@@ -207,6 +208,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
     GOOD_ENOUGH_THRESH = 0.005
 
     ndim = image.ndim
+    anisotropic = radius[1:] != radius[:-1]
```

Comment:
That does satisfy my paranoia.

Sorry if it feels like I am giving you a hard time about this PR.  I am very appreciative of all the work you have put in.